stable/weave-cloud/templates/flux.yaml

```diff
@@ -61,6 +117,16 @@ items:
             - name: agent
               args:
                 - '--token={{.Values.ServiceToken}}'
-              image: 'quay.io/weaveworks/fluxd:0.3.0'
+                - '--connect=wss://cloud.weave.works/api/flux'
+                - >-
+                  --memcached-hostname=weave-flux-memcached.kube-system.svc.cluster.local
```

Comment:
Should this be a variable?

Reply:
No, because we bundle memcached here, using some another memcached instance elsewhere doesn't make much sense.